zebra/zserv.c

```diff
@@ -1042,6 +1043,41 @@ int zsend_route_notify_owner(struct route_entry *re, struct prefix *p,
 	return zebra_server_send_message(client);
 }
 
+void zsend_rule_notify_owner(struct zebra_pbr_rule *rule,
+			     enum zapi_rule_notify_owner note)
+{
+	struct listnode *node;
+	struct zserv *client;
+	struct stream *s;
+
+	zlog_debug("%s: Notifying %u", __PRETTY_FUNCTION__, rule->unique);
```

Comment:
Don't we need to guard the zlog_debug's from this function?

Reply:
Fixing this